app/models/friendship.rb

```diff
@@ -0,0 +1,41 @@
+class Friendship < ApplicationRecord
+  belongs_to :user
+  belongs_to :friend, class_name: "User"
+  validates :friend_id, uniqueness: { scope: :user_id, message: "Already Sent" }
+  scope :friends, -> {where(status: true)}
+  #scope :friends, -> {where(friend_id: User.current.id).or(where( user_id: User.current.id)).and(where(status: true))}
+
+  def exist_friend_id
```

Comment:
naming conventions needs to be improved for both method name & error description